components/LeagueEntries/LeagueEntries.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { Button } from '../Button/Button';
+import { cn } from '@/utils/utils';
+import { EntryStatus } from '../EntryStatus/EntryStatus';
+import { ILeagueEntriesProps } from './LeagueEntries.interface';
+import React from 'react';
+
+/**
+ * A card that contains information on the user's entry for this league. Contains the entry number, their entry status (alive or eliminated), team logo once a pick is set, and a button to make a pick or change their pick
+ * @param props - the props for LeagueEntries
+ * @param props.entryNumber - the entry number of the user in that specific league
+ * @param props.isEliminated - If true, the user is flagged as eliminat4ed
+ * @param props.isPickSet - if true, the team logo of the picked team shows up on the LeagueEntries card and the button changes from "make a pick" to "chagne pick"
+ * @returns {React.JSX.Element} - A div element that contains the user's entry information
+ */
+const LeagueEntries = ({
+  entryNumber,
+  isEliminated = false,
+  isPickSet = false,
+}: ILeagueEntriesProps): React.JSX.Element => (
```

Comment:
Update to use the explicit import of JSX.
```suggestion
}: ILeagueEntriesProps): JSX.Element => (
```

Reply:
Can you explain one one is prefered over the other? I get the sense that `React.JSX.Element` might be the older way of doing things but don't want to assume.

Reply:
Updated in latest commit.